Makefile

```
@@ -41,6 +42,7 @@ api-generate: $(OPENAPI_GENERATOR) clean-openapi $(API_SPEC_PATH)
 		--ignore-file-override=$(API_SERVICE_PATH)/.openapi-generator-ignore \
 		--config=$(API_SERVICE_PATH)/.openapi-config.yml \
 		-DmodelTests=false,modelDocs=false,apiTests=false,apiDocs=false;
+	[ "$(old_spec)" != "$(API_SPEC_PATH)" ] && rm -f $(old_spec) || echo "used existing definition";
```

```suggestion
	[ "$(old_spec)" != "$(API_SPEC_PATH)" ] && rm -f $(old_spec)
```
(the file name can match but the timestamp can be newer if you edited it by hand, so not always "used existing definition").